docs/components/laser-controls.md

```diff
@@ -56,9 +56,10 @@ to the distance to the intersection point.
 
 ## Properties
 
-| Properties | Description        |
-|------------|--------------------|
-| hand       | `left` or `right`. |
+| Properties | Description                                         | Default |
+|------------|-----------------------------------------------------|---------|
+| hand       | `left` or `right` when two controllers are present. | `right` |
+| model      | Whether the default controller models are loaded.   | `true`  |
```

Comment:
The position of the laser is model specific and letting users override the model won't play gracefully

Reply:
It looks like there was some effort put into allowing the model to specify its own offset
[original](https://github.com/aframevr/aframe/blob/master/src/components/laser-controls.js#L43)/[proposed](https://github.com/aframevr/aframe/pull/3138/files#diff-3f66fed14508fae90dc58b66fac99025R51)